sb_vision/calibration/cli.py

```diff
@@ -64,8 +64,7 @@ def main(args=None):
 
     calibration = fit(training_examples)
 
-    if 'marker_size' in config_data:
-        calibration['marker_size'] = config_data['marker_size']
+    calibration['marker_size'] = config_data['marker_size']
```

Comment:
This will error for config file version 1. We should either alert about that when we check the version number (i.e: not support version 1) or allow it here if (though only if) we're reading a version 2 config.

Reply:
fixed in ad62c79